oshi-core/src/main/java/oshi/driver/windows/wmi/Win32PhysicalMemory.java

```diff
@@ -24,14 +24,14 @@ public final class Win32PhysicalMemory {
      * Physical Memory properties for Win10 and later.
      */
     public enum PhysicalMemoryProperty {
-        BANKLABEL, CAPACITY, SPEED, MANUFACTURER, SMBIOSMEMORYTYPE
+        BANKLABEL, CAPACITY, SPEED, MANUFACTURER, PARTNUMBER, SMBIOSMEMORYTYPE
     }
 
     /**
      * Physical Memory properties for Win8 and earlier.
      */
     public enum PhysicalMemoryPropertyWin8 {
-        BANKLABEL, CAPACITY, SPEED, MANUFACTURER, MEMORYTYPE
+        BANKLABEL, CAPACITY, SPEED, MANUFACTURER, MEMORYTYPE, PARTNUMBER
```

Comment:
Just curious why the order is different between the Win10+ and Win8- versions?  Doesn't matter for the code, just odd to see it reversed.

Reply:
Becasue of alphabetical order, in w10:
PARTNUMBER, SMBIOSMEMORYTYPE
in w8:
MEMORYTYPE, PARTNUMBER

There is no MEMORYTYPE in w10, instead we have SMBIOSMEMORYTYPE